src/table/table.component.tsx

```diff
@@ -435,12 +437,14 @@ const Table = React.memo((props: TableProps): React.ReactElement => {
             })}
             {/* Need to make this a tr with a td column with the correct colSpan 
                     to be a valid HTML table */}
-            {/* eslint-disable-next-line jsx-a11y/role-supports-aria-props */}
+            {}
             <Backdrop
               component="tr"
+              // At the moment component isn't working: https://github.com/mui/material-ui/issues/44283
+              slots={{ root: 'tr' }}
               sx={{ position: 'absolute', zIndex: 100, height: 'inherit' }}
               open={!loadedData}
-              role="none"
+              // role="none"
```

Comment:
Why the change to comment out the `role="none"`? I presume this was needed at some point to ensure that it counted as a valid HTML table without messing up the accessibility roles. If this is fine now - remove the `role="none"` comment and also the empty `{}`?

One thing I note is that the backdrop no longer looks like a backdrop - instead it is just the loading spinner. Also our custom styles aren't applying due to this workaround. I think we should try and get the MUI issue re-opened by providing a minimal example.

I don't think this is a blocking issue as the spinner still works, just doesn't look as nice as the previous one...